### PR TITLE
Feat/message board

### PR DESCRIPTION
**Pull request by tundealabi**
- redirect to the messaging board page by default
- fetch a default room if none is passed
- import and use message board from zuri UI

### CODE REVIEW

File: frontend/root-config/src/index.ejs [hunk 1]
```diff
@@ -53,7 +53,8 @@
         "imports": {
           "single-spa": "https://cdn.jsdelivr.net/npm/single-spa@5.9.0/lib/system/single-spa.min.js",
           "vue": "https://cdn.jsdelivr.net/npm/vue@2.6.11/dist/vue.min.js",
-          "vue-router": "https://cdn.jsdelivr.net/npm/vue-router@3.1.6/dist/vue-router.min.js"
+          "vue-router": "https://cdn.jsdelivr.net/npm/vue-router@3.1.6/dist/vue-router.min.js",
```
Comment (zxenonx) [inline frontend/root-config/src/index.ejs:56]: @tundealabi, please why are we importing vue and vue-router?

Comment (bori7) [reply frontend/root-config/src/index.ejs:56]: Same thoughts exactly.

Isn't this SPA a react app?

File: frontend/zc_messaging/src/utils/fetchDefaultRoom.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const fetchDefaultRoom = async (orgId, userId) => {
+  const response = await axios.get(
+    `https://chat.zuri.chat/api/v1/sidebar?org=${orgId}&user=${userId}`
+  )
+  const { public_rooms } =
+    response.data?.data?.find(room => room.name === "Channels") || {}
+  const defaultRoom =
+    public_rooms?.find(room => room.room_name === "general") || {}
+  return {
+    room_id: defaultRoom.room_id
```
Comment (bori7) [inline frontend/zc_messaging/src/utils/fetchDefaultRoom.js:12]: The case for defaultRoom if its null, doesn't have a room_id-> {}

File: frontend/root-config/src/index.ejs (hunk 1 above)
Comment (tundealabi) [reply frontend/root-config/src/index.ejs:56]: dunno.. didn't add that

File: frontend/zc_messaging/src/utils/fetchDefaultRoom.js (hunk 2 above)
Comment (tundealabi) [reply frontend/zc_messaging/src/utils/fetchDefaultRoom.js:12]: won't throw an error, will just be undefined

File: frontend/root-config/src/index.ejs (hunk 1 above)
Comment (zxenonx) [reply frontend/root-config/src/index.ejs:56]: I think it was @benjamin-bala, maybe for when a Vue app will be integrated...

**Pull request merged**